sys/include/net/netstats.h

```diff
@@ -41,6 +50,16 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief   Get a string description for @ref netstats_module_t, for debugging
```

Comment:
>would exceed the 80 character limit and would require a linebreak (which is not simple with doxygen)

since when? AFAIK, you can just put in a line break and move the text for the second line again to the correct indentation level. No need for magic.